nornir/plugins/tasks/text/template_string.py

```diff
@@ -2,13 +2,14 @@
 from nornir.core.task import Result
 
 
-def template_string(task, template, jinja_filters=None, **kwargs):
+def template_string(task, template, jinja_filters=None, path=None, **kwargs):
     """
     Renders a string with jinja2. All the host data is available in the tempalte
 
     Arguments:
         template (string): template string
         jinja_filters (dict): jinja filters to enable. Defaults to nornir.config.jinja_filters
+        path (string): path that to the folder that the template extends. Defaults to None
```

Comment:
We will need to rephrase this. Something like `path to folder with additional jinja2 templates and resources`.

Reply:
So it is actually the path that the templates extends.. so maybe base_template_path? it is too long, so happy for suggestions

Reply:
So this enables more than just the `extend` as in your use case. It also enables the imports and includes, it's basically a path to jinja2 assets. In any case, I think it's best to document as `if set, load FileSystemLoader with this path`.

Reply:
You mean in the comment? sure NP